stl/inc/type_traits

```diff
@@ -521,6 +410,43 @@ _INLINE_VAR constexpr bool is_volatile_v<volatile _Ty> = true;
 template <class _Ty>
 struct is_volatile : bool_constant<is_volatile_v<_Ty>> {};
 
+// STRUCT TEMPLATE is_function
+template <class _Ty>
+_INLINE_VAR constexpr bool is_function_v =
+    !is_const_v<const _Ty> && !is_reference_v<_Ty>; // determine whether _Ty is a function

```

Comment:
this comment is a bit redundant, I would _hope_ that is_function_v determines whether _Ty is a function.

Perhaps change this comment to describe why !is_const_v<const _Ty> && !is_reference_v<_Ty> implies _Ty is a function, or just remove the comment

Reply:
Done